Translations/fr.po

```diff
@@ -8476,11 +8477,11 @@ msgid ""
 "I'll put it bluntly - kill him before he kills you and adds your corpse to "
 "his collection."
 msgstr ""
-"J'en sais plus que vous ne le pensez sur cet horrible démon. Ses petits amis "
-"m'ont attrapé et ont réussi à me prendre ma jambe avant que Griswold ne me "
-"tire hors de ce trou.\n"
+"J'en sais plus que vous ne pensez sur cette horreur. Ses petits copains "
+"m'avaient attrapé et avaient réussi à me niquer la jambe avant que Griswold "
```

Comment:
"niquer la" sounds too familiar and recent

Reply:
Same remark as bizness. I tried to express the anger of Wirt about this matter. I thought about what I could say in his stead, and "niquer" popped naturally. Alternatives would be "bousiller", "flinguer", and so on, but "niquer" had my preference.